scripts/composer/ScriptHandler.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * @file
+ * Contains \DrupalProject\composer\ScriptHandler.
+ */
+
+namespace DrupalProject\composer;
+
+use Composer\Script\Event;
+use Symfony\Component\Filesystem\Filesystem;
+
+class ScriptHandler {
+
+  protected static function getDrupalRoot($project_root) {
+    return $project_root .  '/web';
```

Comment:
Yeah drupal-scaffold has this feature already. I think we could detect it via composer installers path or add something link "drupal-root-dir" to the composer.json. A own subclass of ScriptHandler should work as well ...